docs/examples/zoom.html

```diff
@@ -0,0 +1,151 @@
+            target="_blank" class="btn btn-default">View source code</a>
+        </div>
+      </div>
+    </div>
+
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <div>
+          <label>Type :</label> <select style="width: 200px;" id="zoom_type">
+            <option value="lat/lng">lat/lng</option>
+            <option value="country" selected>country</option>
+          </select>
+        </div>
+        <div id="country" style="margin-top:5px;" >
+          <label>Country :</label> <select style="width: 200px;" id="country_type">
+            <option value="IND" selected>India</option>
+          </select>
+        </div>
+        <div id="lat/lng" style="margin-top:5px;display: none;">
+          <div>
+            <label for="lng">Longitude : </label>
+         <input id="lng" value="85.37" type="number" placeholder="Longitude" />
+        </div>
+        <div>
+          <label for="lat" >Latitude : </label>
+         <input id="lat" value="27.7" type="number" placeholder="Latitude" />
+        </div>
+        <div>
+          <label for="scale">Scale : </label>
+         <input id="scale" value="26" type="number" placeholder="Scale" />
+        </div>
+        </div>
+      </div>
+    </div>
+    <div id="vizContainer"></div>
+  </div>
+  <script>
+    let zoomType = document.getElementById('zoom_type').value;
+    let countryType = document.getElementById('country_type').value;
+    let lat = document.getElementById('lat').value;
+    let lng = document.getElementById('lng').value;
+    let scale = document.getElementById('scale').value;
+
+    function getScaleValues(){
+      if(document.getElementById('zoom_type').value === 'country'){
+        // return scale options for country 
+        let options = {scaleType:'country',values:{iso_a3:document.getElementById('country_type').value}};
+        return options;
+      } else {
+        // return scale options for lat/lng 
+        let options = {scaleType:'lat/lng',values:{latitude:document.getElementById('lat').value,longitude:document.getElementById('lng').value,scale:document.getElementById('scale').value}};
+        return options;
+      }
+    }
+
+    var viz = d3.maptable('#vizContainer')
+      .csv('basic.csv')
+      .map({
+        path: 'countries.json',
+        markers: {},
+        saveState:true,
+        zoom:true,
+        countries:{
+          attr:{
+            "stroke-width": 1
+          },
+        },
+        // scale options template for latitude/longitude based zoom
+        // defaultScaleTo: {scaleType:"lat/lng",values:{latitude:'27.7',longitude:'85.37',scale:32}}
+        // scale options template for country based zoom
+        // defaultScaleTo: {scaleType:'country',values:{iso_a3:'CHN',fillColor:"#EEEEEE"}}
+        // getScaleValues() function returns scale options for this example page, you can directly pass an options object insted of function
+        defaultScaleTo: getScaleValues
```

Comment:
defaultScaleTo was defined on the readme as an object that tapes `scaleType` and `values`. But here the `getScaleValues` is a function

Reply:
Yes, 'defaultScaleTo' accepts both object as well as function. Updated on readme now.